frontend/src/util/DateOps.tsx

```diff
@@ -1,7 +1,7 @@
 import { format } from "date-fns";
 
 export function externalToInternalDate(date: string) {
-  return new Date(date.replace("-", "/"));
+  return new Date(date);
 }
 
 export function internalToExternalDate(date: Date) {
```

Comment:
I also read that .toISOString() also throws errors, can you check that the instance where it is used at is working, such as buyback add, sales report generate, and such items.

Reply:
Both of those things work

Reply:
The issue was that when forming the date object, it was always "Invalid Date", so none of our pages would load. Everything seems to work properly now